packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -293,11 +298,11 @@ export default mixins(
 
   methods: {
     emitInput (newInput: string) {
-      if (this.range && this.value) {
-        if (this.value.length !== 1) {
+      if (this.range) {
+        if (this.multipleValue.length !== 1) {
           this.$emit('input', [newInput])
         } else {
-          const output = [...this.value, newInput]
+          const output = [this.multipleValue[0], newInput]
```

Comment:
is it possible for **multipleValue** to be 0 and if so is **undefined** an acceptable value for **output** on 305?

Reply:
not possible, in this branch of `if` length of multipleValue is always equal 1